NewsFeed/views/CCBRNewsViewController.m

```diff
@@ -190,6 +202,14 @@ - (CGSize)collectionView:(UICollectionView *)collectionView layout:(UICollection
 - (IBAction)didTapButton:(UIButton *)sender {
     if (sender == self.settingsButton) {
         // TODO: Show Settings screen
+        CCBRSettingViewController * settingVC = [[CCBRSettingViewController alloc] init];
```

Comment:
Please move "navigation" logic to the coordinator! View/ViewController should be as dump as possible. They shouldn't know how to construct and present the SettingViewController.